src/sentry/static/sentry/less/components/button.less

```diff
@@ -0,0 +1,87 @@
+@import url("../palette.less");
```

Comment:
are there concerns with this not being the same css as legacy buttons? (btn -> button). that might affect plugins/getsentry